src/main/java/bank/BankServiceMediator.java

```diff
@@ -0,0 +1,35 @@
+package bank;
+
+import bank.clerk.*;
+import lombok.RequiredArgsConstructor;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@RequiredArgsConstructor
+public class BankServiceMediator{
+    private Map<Integer, Clerk> clerks;
```

Comment:
```suggestion
    private final Map<Integer, Clerk> clerks = new HashMap<>();
```
위 코드로 바꿔 보면 어떨까요? final 키워드를 사용하면 의도를 명시적으로 나타낼 수 있을 것 같습니다